src/cron/hourlyNotifications.ts

```diff
@@ -24,7 +26,7 @@ const cron: Cron = {
   handler: async (con, logger) => {
     const personalizedDigestQuery = con
       .createQueryBuilder()
-      .select('upd.*, u.timezone')
+      .select('upd.*, u.timezone, u.weekStart')
```

Comment:
think it needs `u."weekStarts"` due to casing

Reply:
Not sure, but I added just to be safe